Lib/_pyrepl/utils.py

```diff
@@ -2,12 +2,56 @@
 import unicodedata
 import functools
 
+from idlelib import colorizer
+from typing import cast, Iterator, Literal, Match, NamedTuple, Pattern, Self
```

Comment:
Here are some deprecated aliases. Although there are no issues at present, it would be better to switch to `collection.abc`

https://docs.python.org/3.14/library/typing.html#typing.Iterator